UI/index.html

```diff
@@ -0,0 +1,434 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link href="styles.css" rel="stylesheet" charset="utf-8">
+    <meta charset="utf-8">
+    <title>SocNet</title>
+  </head>
+  <body>
+
+    <header class="header">
```

Comment:
It's better to align items vertically, but not to set each of them `top` property.